### ISSUE TITLE
Re-enable disabled jsx-a11y eslint rules

### ISSUE THREAD

**Issue by gpbl** [gpbl/react-day-picker#387]
[jsx-a11y/no-noninteractive-element-interactions](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md) and [jsx-a11y/interactive-supports-focus](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/interactive-supports-focus.md) are disabled in .eslintrc for no good reasons. They should be enabled again and code should be fixed.
